torchtext/experimental/vocab.py

```diff
@@ -71,6 +71,23 @@ def vocab_from_file(file_object, min_freq=1, unk_token='<unk>', num_cpus=4):
     return Vocab(vocab_obj)
 
 
+def build_vocab_from_iterator(iterator):
+    """
+    Build a Vocab from an iterator.
+
+    Arguments:
+        iterator: Iterator used to build Vocab. Must yield list or iterator of tokens.
+    """
+
+    counter = Counter()
```

Comment:
If we don't expose `min_freq` to `vocab`, then this function effectively becomes the same thing as
```python
ordered_dict = OrderedDict([(v, k) for k, v in enumerate(sorted(set(iterator)))])
```

Reply:
The `iterator` will yield repeated tokens so we need a counter first and rank the high frequency tokens first. `set` will remove the frequency of the tokens.